queries.sql

```diff
@@ -15,3 +15,59 @@ SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3 ;
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified' ;
+
+SELECT name, species from animals;
+
+ROLLBACK;
+
+SELECT name, species from animals;
+
+BEGIN;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
+
+SELECT name , species from animals;
+
+BEGIN;
+
+DELETE FROM animals;
+
```

Comment:
- [ Required ] I think it would be nice if you made a `SELECT` query to verify that indeed all the animals were deleted before making the `ROLLBACK`.